moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group.h

```diff
@@ -116,18 +116,23 @@ class MoveGroup
       \param opt. A MoveGroup::Options structure, if you pass a ros::NodeHandle with a specific callback queue, it has to be of type ros::CallbackQueue
         (which is the default type of callback queues used in ROS)
       \param tf. Specify a TF instance to use. If not specified, one will be constructed internally.
-      \param wait_for_server. Optional timeout for connecting to the action server. If it is not specified, the wait time is unlimited.
+      \param wait_for_servers. Timeout for connecting to action servers. Zero time means unlimited waiting.
     */
   MoveGroup(const Options &opt, const boost::shared_ptr<tf::Transformer> &tf = boost::shared_ptr<tf::Transformer>(),
-            const ros::Duration &wait_for_server = ros::Duration(0, 0));
+            const ros::WallDuration &wait_for_servers = ros::WallDuration());
```

Comment:
Nope. This timeout is about getting the action server fired up. No ROS-time involved at all. We definitely should use wall time, otherwise (on a crazy fast machine) 10s might become a single second...